obabeliface/obabeliface.cpp

```diff
@@ -63,12 +66,12 @@ namespace Molsketch
 
   QString number2symbol( int number )
   {
-    return eTable.GetSymbol(number);
```

Comment:
Missing removal of `OpenBabel::OBElementTable eTable ;`

Reply:
Shouldn't The corresponding {} braces be removed also?

Reply:
Just pick my patch from pkgsrc-wip as is.

Reply:
https://github.com/NetBSD/pkgsrc-wip/blob/master/molsketch/patches/patch-obabeliface_obabeliface.cpp